### PR TITLE
Clarify Trello boards and getting started info, close #26

### LINKED ISSUE #26: Add Trello page
- [x] How to copy boards (https://help.trello.com/article/802-copying-cards-lists-or-boards)
- [x] How to use checklists [<- did a bit about this in the current 'getting started' page.]

☝️ Is there an existing document describing how to 'use' checklists? I thought I remembered that there is one.
[[This workflow document](https://docs.google.com/document/d/1bgT2GBGZrENlrQ5uugY_3GFpeqmt1urHYNCuqCYuATw/edit#heading=h.1zjeprm4su2m) does a bit of explaining]

### PR DESCRIPTION
**Pull request by plocket**
I think this shouldn't affect the bootcamp, but I want to make sure it meets with approval.

### CODE REVIEW

File: docs/pdfs.md [hunk 1]
```diff
@@ -6,7 +6,9 @@ slug: /pdfs
 ---
 
 ## Add labels and fields
-We recommend using [the Documate online free PDF editor](https://www.documate.org/pdf). It can do everything that AssemblyLine makes use of as far as PDF labels and fields go. Documate has [a page that talks about its PDF editor](https://documate.wrkflows.io/).
+We recommend you use [specific names for labels](docs_vars_reference.md) to take full advantage of the tools we've created.
```
Comment (BryceStevenWilley) [inline docs/pdfs.md:9]:
```suggestion
We recommend you use [specific names for labels](doc_vars_reference.md) to take full advantage of the tools we've created.
```

**Review by BryceStevenWilley — APPROVED** [review]
Accept the suggested change to fix a broken link, but otherwise LGTM

File: docs/pdfs.md (hunk 1 above)
Comment (plocket) [reply docs/pdfs.md:9]: Awesome! How the heck did you find that?!

Comment (BryceStevenWilley) [reply docs/pdfs.md]: I didn't actually, it was docusaurus! I ran `npm run start`, and showed me this error:
```

✔ Client
  Compiled successfully in 582.98ms

warn Docs markdown link couldn't be resolved: (docs_vars_reference.md) in /home/brycew/Developer/LITLab/docassemble-AssemblyLine-documentation/docs/pdfs.md for version current


```

**Pull request merged**